source/includes/_tipser-react-elements.md

```diff
@@ -251,3 +251,62 @@ If you'd like to unify our design with your own color-theme, you can use our pri
 }
 ```
 If you'd like to change other elements' color as well, please use specific classes to override the styles.
+
+## Server side rendering
+
+If you want to render Tipser-Elements' components on the server side, you can do it in three steps:
+1. At the beginning you have to know ids of products and collections that you want to render on specific url.
+2. Before rendering you have to prepare the state which contains the data necessary to render specific components. 
+It is easy to build this state - our lib includes the tool to do it!
+3. The state you have to apply to `TipserElementsProvider` as `initialState` prop
```

Comment:
The difference between point 3 and 4 is not clear. Should we try to be more specific? I guess point 3 is about doing it on a server and point 4 on the frontend?

Reply:
Clear for me, to be honest. Could you provide specific change suggestion @piotr-sobczyk ?. 

in point 4. i think @mateuszsikora did not put details how to transfer state on the frontend side, because that is very specific to their own custom implementation.

Reply:
Okay, if it's clear for you guys then forget about my comment.